ghedesigner/radial_numerical_borehole.py

```diff
@@ -171,18 +152,14 @@ def fill_radial_cell(self, radial_cell, resist_p_eq, resist_f_eq, resist_tg_eq):
                 ],
                 dtype=np.double,
             )
-        cell_summation += num_conv_cells
-
-        # TODO: verify whether errors are possible here and raise exception if needed
-        # assert cell_summation == (num_fluid_cells + num_conv_cells)
+        cell_summation += self.num_conv_cells
 
         # load pipe cells
-        for idx in range(cell_summation, num_pipe_cells + cell_summation):
-            j = idx - cell_summation
-            inner_radius = self.r_in_tube + j * self.thickness_pipe
-            center_radius = inner_radius + self.thickness_pipe / 2.0
-            outer_radius = inner_radius + self.thickness_pipe
-            conductivity = log(self.r_borehole / self.r_in_tube) / (TWO_PI * resist_p_eq)
+        for j, idx in enumerate(range(cell_summation, self.num_pipe_cells + cell_summation)):
+            inner_radius = self.r_in_tube + j * self.thickness_pipe_cell
+            center_radius = inner_radius + self.thickness_pipe_cell / 2.0
+            outer_radius = inner_radius + self.thickness_pipe_cell
+            conductivity = log(self.r_borehole / self.r_in_tube) / (TWO_PI * resist_pg_eq)
```

Comment:
This was a bug. The resistance value being used here was the pipe resistance, but should have been the effective resistance of the pipe and grout combined.